### ISSUE TITLE
Geyser Lizard not spawning may be due to geysers not erupting. 

### ISSUE THREAD

**Issue by Wanpide** [EdenServer/community#6018]
### Checklist

- [ ] I have searched for existing issues for issues like this one. The issue has not been posted.
- [ ] I have provided reproducible steps. (No "as the title says" posts please. ***Please try to state facts only, and avoid assumptions/opinions.***)
- [ ] I have provided my client version in the details. (type /ver into your game window)
- [ ] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details

Spent several hours in Dangruf Wadi on THF/RNG using wide scan to search the area for Geyser Lizard w/o seeing one pop. I have not seen the geysers erupt either which may be the reason the NM is not spawning. I saw this issue was supposed to be resolved Dec 2023 but I think it is broken again.

**Comment by hyakuzaku** [comment]
![image](https://github.com/user-attachments/assets/e02404b8-f39d-4dbf-b08e-27aaeb717c86)

At some point a very long time ago, I found it in the circled area here after a server maintenance.

Pretty sure because there is no accurate capture of the zone since it changed in 75+ era everything is messed up in there, mob names & the opo-opos, etc.

Similarly, I would like to know about Orctrap in Carpenter's Landing as I've spent hours THF/RNG trying to pop it and obviously the PH or spawn area is shifted.
